keyboards/canary/canary60rgb/v2/readme.md

```diff
@@ -0,0 +1,26 @@
+# Canary60RGB_V2
+
+![canary60rgb_v2_1](https://github.com/AnthonyNguyen168/keyboards/blob/main/4.jpg)
+![canary60rgb_v2_2](https://github.com/AnthonyNguyen168/keyboards/blob/main/5.jpg)
```

Comment:
These (and the other) images should be rehosted on Imgur or a similar image hosting site, rather than github.

Reply:
agree, I will change in the next commit.